src/poker/EquityEvaluator.java

```diff
@@ -112,41 +110,49 @@ public List<Double> calculateEquityTest(List<Card> hand1, List<Card> hand2) {
         return finalCalculateEquity(gameResult);
     }
 
-    public void addPlayerWinsTest(List<Card> possibleBoard, GameResult gameResult, List<Card> hand1, List<Card> hand2) {
+    public String addPlayerWinsTest(List<Card> possibleBoard, List<Card> hand1, List<Card> hand2) {
         ResultHandOut resultPlayer1 = winningHand(hand1, possibleBoard);
         ResultHandOut resultPlayer2 = winningHand(hand2, possibleBoard);
 
         if (resultPlayer1.getValue() > resultPlayer2.getValue()) {
-            gameResult.increasePlayer1Wins(1);
+            System.out.print("Player1 wygrał z " + resultPlayer1.getBestHandValues());
+            System.out.println(" Player2 przegrał z " + resultPlayer2.getBestHandValues());
+            return "Player1";
 
         } else if (resultPlayer1.getValue() < resultPlayer2.getValue()) {
-            gameResult.increasePlayer2Wins(1);
+            System.out.print("Player2 wygrał z " + resultPlayer1.getBestHandValues());
+            System.out.println(" Player1 przegrał z " + resultPlayer2.getBestHandValues());
+            return "Player2";
 
         } else {
             List<Integer> hand1Cards = resultPlayer1.getBestHandValues();
             List<Integer> hand2Cards = resultPlayer2.getBestHandValues();
 
             for (int y = 0; y < hand1Cards.size(); y++) {
                 if (hand1Cards.get(y) > hand2Cards.get(y)) {
-                    gameResult.increasePlayer1Wins(1);
-                    break;
+                    System.out.print("Player1 wygrał z " + resultPlayer1.getBestHandValues());
+                    System.out.println(" Player2 przegrał z " + resultPlayer2.getBestHandValues());
+                    return "Player1";
```

Comment:
wez zmien to na ENUMa bo serio lipa jest zwracac takiego stringa 
enum w stylu WIN_PLAYER1, WIN_PLAYER2, DRAW